src/Sulu/Bundle/PageBundle/Resources/config/forms/page_block_settings.xml

```diff
@@ -25,5 +25,31 @@
 
             <tag name="sulu.block_setting_icon" icon="su-hide" />
         </property>
+
+        <section name="segment" visibleCondition="__webspace.segments.length &gt; 0">
+            <properties>
+                <property name="segment_enabled" type="checkbox">
+                    <params>
+                        <param name="description">
+                            <meta>
+                                <title>sulu_page.segment_description</title>
+                            </meta>
+                        </param>
+                        <param name="icon" value="su-focus" />
+                        <param name="label">
+                            <meta>
+                                <title>sulu_page.segment_label</title>
+                            </meta>
+                        </param>
+                        <param name="skin" value="heading" />
+                        <param name="type" value="toggler" />
+                    </params>
+
+                    <tag name="sulu.block_setting_icon" icon="su-focus" />
+                </property>
+
+                <property name="segment" type="segment_select" visibleCondition="segment_enabled == true" />
```

Comment:
would be nice to hide the `none selected` option here 🙂

Reply:
but we can do that in the article pr

Reply:
Hm, talked to @chirimoya, and we said that this is good enough... The thing is that the default segment should probably be selected by default, and it would need the webspace for that... But yeah, we can reevaluate in the PR for the article segments, and if it can be done easily we can add it.